package.json

```diff
@@ -45,8 +50,9 @@
     "eslint-plugin-import": "2.10.0",
     "greenkeeper-lockfile": "1.14.0",
     "mime": "2.2.1",
-    "mocha": "5.0.5",
+    "mocha": "mochajs/mocha#88b9882",
```

Comment:
Mocha will support `--exclude` option in the next version.
See https://github.com/mochajs/mocha/pull/3210